### PR TITLE
Fixed iOS input box compatibility

### LINKED ISSUE #3: iOS input box compatibility
Currently when writing in the input form on an iOS device the texts shows partially hidden.

### PR DESCRIPTION
**Pull request by dcondrey**
Resolves #3

### CODE REVIEW

File: src/components/TitleInput/TitleInput.module.css [hunk 1]
```diff
@@ -62,7 +62,7 @@ input {
   border-bottom: 1px solid black;
 
   width: 100%;
-  height: 40px;
+  height: inherit;
```
Comment (Deohgu) [inline src/components/TitleInput/TitleInput.module.css:65]: The initial idea behind `height: 40px` was to have the text sit on the `input` bottom border and also have the input be similar in size to the surrounding text, with `height: inherit` it no longer does so.
At least not on the browser, haven't had the chance to test elsewhere.

**Review by Deohgu — CHANGES_REQUESTED** [review]

File: src/components/TitleInput/TitleInput.module.css (hunk 1 above)
Comment (dcondrey) [reply src/components/TitleInput/TitleInput.module.css:65]: Can you post screenshot as well as screenshot for desired effect?  

Here are browser tests for MacOS 11.1 using Safari 14.0.1 with and witihout height adjustment.  As well as a test with Android 7.1 using default browser.  https://imgur.com/a/2I2F6W9

screenshots

**Comment by dcondrey** [comment]
I tried many different things this evening but I have not been able to get Safari to match the way it looks in Android except if you were to do something like this:

```html
<div class="TitleInput_container__1aLA0">
    <h1 class="TitleInput_title__20RCS">WITHIN THE WIKIPEDIA CATEGORY OF</h1>
    <style>[contenteditable]:focus { outline: 0px solid transparent; }</style>
    <h1 id="inputText" class="TitleInput_title_20RCS" contenteditable="" style="border-bottom: 1px solid black;text-transform:uppercase;">test</h1>
    <h1 class="TitleInput_title__20RCS">YOU WILL BE PROVIDED A Random   ARTICLE.</h1>
</div>
```

See [npm install react-contenteditable](https://www.npmjs.com/package/react-contenteditable)

But that is more of a major change and I just started learning react so I'll leave you to it..

**Comment by Deohgu** [comment]
It is working on the example provided but it is likely because it is an H1 element.
I'll have a look when I finish refactoring the whole code the whole thing is still very messy. 😅
[Here a CSS reset](https://stackoverflow.com/questions/12093540/how-can-i-make-my-css-code-compatible-with-all-browsers) is advised but the answer is 8 years old so it might not be a modern approach. I will also be adding Styled-Components to the project which also injects CSS to make it compatible across browsers so hopefully that helps.
Thank you @dcondrey.

**Pull request closed without merging**